test/Worker/Grpc.Tests/DependencyInjection/DurableTaskBuilderExtensionsTests.cs

```diff
@@ -1,9 +1,17 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using Grpc.Core;
 using Microsoft.Extensions.DependencyInjection;
 
+#if NET6_0_OR_GREATER
+using Grpc.Net.Client;
+#endif
+
+#if NETFRAMEWORK
+using Grpc.Core;
+using GrpcChannel = Grpc.Core.Channel;
```

Comment:
Is this something that global usings support? I'm wondering if there is a simple way we can reduce some of this duplication.

Reply:
Yeah! let me try that.